Snakefile

```diff
@@ -420,92 +637,1344 @@ rule atlas_experiment_summary:
+        echo "ISL dir: $expIslDir"
+
+        [ ! -z $expIslDir+x ] || (echo "snakemake param exp_isl_dir needs to defined in rule" && exit 1)
 
-        [ ! -z $expIslDir+x} ] || (echo "snakemake param exp_isl_dir needs to defined in rule" && exit 1)
+        if [[ "{wildcards.metric}" == "tpms" ]]; then
+            metrictype="tpm"
+        else
+            metrictype="fpkm"
+        fi
 
-        if [ -s "$expIslDir/genes.{wildcards.metric}.htseq2.tsv" ]; then
-            # maybe rsync could be better here?
-            cp $expIslDir/genes.{wildcards.metric}.htseq2.tsv {output.normalised_counts_undecorated}
-        elif [ -s "$expIslDir/genes.{wildcards.metric}.featurecounts.tsv" ]; then
-            cp $expIslDir/genes.{wildcards.metric}.featurecounts.tsv {output.normalised_counts_undecorated}
+        if [ -s "$expIslDir/genes.$metrictype.htseq2.tsv" ]; then
+            rsync -avz $expIslDir/genes.$metrictype.htseq2.tsv {output.normalised_counts_undecorated}
+        elif [ -s "$expIslDir/genes.$metrictype.featurecounts.tsv" ]; then
+            rsync -avz $expIslDir/genes.$metrictype.featurecounts.tsv {output.normalised_counts_undecorated}
         else
-            echo "$expIslDir/genes.{wildcards.metric}.htseqORfeaturecounts.tsv not found"
+            echo "$expIslDir/genes.$metrictype.htseqORfeaturecounts.tsv not found"
             exit 1
         fi
         """
 
 
 rule copy_transcript_files_from_isl:
+    """
+    This rule attemps to copy Kallisto TPM transcripts if metrics 'tpms' exists.
+    If file does not exist for an accession, this rule can be skipped.
+    """
+    log: "logs/{accession}-copy_transcript_files_{metric}_from_isl.log"
     params:
-        exp_isl_dir=get_isl_dir()
+        organism=get_organism(),
+        isl_dir=get_isl_dir()  
     output:
-        transcripts="{accession}-transcripts-{metric}s.tsv.undecorated"
+        transcripts="{accession}-transcripts-{metric}.tsv.undecorated"
     shell:
         """
-        if [ -s "{params.exp_isl_dir}/transcripts.{wildcards.metric}.kallisto.tsv" ] ; then
-            # maybe rsync could be better here?
-        	cp {params.exp_isl_dir}/transcripts.{wildcards.metric}.kallisto.tsv {output.transcripts}
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+        expIslDir={params.isl_dir}/{wildcards.accession}/{params.organism}
+        echo "ISL dir: $expIslDir"
+
+        [ ! -z $expIslDir+x ] || (echo "snakemake param exp_isl_dir needs to defined in rule" && exit 1)
+
+        if [ -s "$expIslDir/transcripts.tpm.kallisto.tsv" ] ; then
+            rsync -avz $expIslDir/transcripts.tpm.kallisto.tsv {output.transcripts}
         else
-        	echo "{params.exp_isl_dir}/transcripts.{wildcards.metric}.kallisto.tsv not found - skipping"
+            echo "$expIslDir/transcripts.tpm.kallisto.tsv not found - skipping"
         fi
```

Comment:
Thanks @pcm32, this is an important point. This code was inherited from [here](https://github.com/ebi-gene-expression-group/atlas-prod/blob/develop/db/scripts/import_baseline_rnaseq_experiment.sh#L39-L51). Should we bring `transcripts.fpkm.kallisto.tsv` to data production? 
Pinning @pinin4fjords in case he wants to comment.

Reply:
I guess that if we have fpkms for transcripts we should bring it over, right? Can you check if we have them in general (I guess in noah) among the ISL results?

Reply:
In any case, if this wasn't part of the original code, I'm happy for us to leave this like this towards closing this monster PR and open an issue and then resolve it in a smaller PR.

Reply:
Issue: https://github.com/ebi-gene-expression-group/bulk-recalculations/issues/32

Reply:
thanks - `transcripts.fpkm.kallisto.tsv` are available, but have not been reported by Atlas so far (see the FTP). I will add a new feature in a different branch to allow this.